### DISCUSSION TITLE
现在绳网的氛围好舒服，要是一直这样就好了希望不要有节奏进来

### DISCUSSION THREAD

**Discussion by gtx-d** [share121/inter-knot#3264] [其他]
希望这种氛围一直保持下去不要想米游社一样

**Comment by yzysrewrwqr** [comment]
整个进站考试，等级回复，卡死一堆水军<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by wyqsagiri** [reply]
同意

**Reply by qmtqyh** [reply]
智齿，但是会登绳网的人素质都不会差到哪去吧

**Reply by LT-UT** [reply]
也不一定，如果是那些带有强烈恶意的水军来了，到时又变得。。。额

**Reply by Alsuhold** [reply]
> 也不一定，如果是那些带有强烈恶意的水军来了，到时又变得。。。额

至少以目前网站原理来说水军肯定进不来

**Comment by Zaiyusi** [comment]
那些脑子有问题的都进不来<img src="https://bbs-static.miyoushe.com/static/2023/04/28/f2f1488ffbfff3fa96d4db6047e5f409_6776930953450600054.png" alt="米仔_不了" width="75" >
